rustls-platform-verifier/Cargo.toml

```diff
@@ -29,7 +29,7 @@ cert-logging = ["base64"]
 docsrs = ["jni", "once_cell"]
 
 [dependencies]
-rustls = { version = "0.22.1", features = ["tls12", "logging"] }
```

Comment:
Was it intentional to drop the `tls12` feature activation?

Reply:
Yep, very intentional. For example, Quinn users will typically not want TLS 1.2 support built. It didn't actually seem to be needed, anyway.

Reply:
Makes sense, just wanted to check explicitly 👍 Thanks!